bin/pfcmd-old.pl

```diff
@@ -1285,9 +1285,9 @@ sub getIptablesTechnique {
 sub stopService {
```

Comment:
We should but not for this release.
We should decide what other commands we should port before just deleting it.